concepts/range-iteration/about.md

```diff
@@ -1,3 +1,6 @@
 # About
 
-TODO: add information on range-iteration concept
+As a programmer you will be required to loop through things. In Go we usually
+iterate [slices](slices), [maps](maps), and [channels](channels).
+
+Within this concept, you will learn to `range` over slices and maps but not channels.
```

Comment:
The `about.md` document is the reference document for students after completing the exercise teaching that concept. As such, it should at least contain all the information from the introduction.

You can read more about this document here: https://github.com/exercism/docs/blob/main/anatomy/tracks/concepts.md#file-aboutmd